yt_dlp/extractor/globo.py

```diff
@@ -80,98 +90,71 @@ class GloboIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        self._request_webpage(
-            HEADRequest('https://globo-ab.globo.com/v2/selected-alternatives?experiments=player-isolated-experiment-02&skipImpressions=true'),
-            video_id, 'Getting cookies')
-
-        video = self._download_json(
-            f'http://api.globovideos.com/videos/{video_id}/playlist',
-            video_id)['videos'][0]
-        if not self.get_param('allow_unplayable_formats') and video.get('encrypted') is True:
-            self.report_drm(video_id)
-
-        title = video['title']
+        info = self._download_json(
+            'https://cloud-jarvis.globo.com/graphql', video_id,
+            query={
+                'operationName': 'getVideoView',
+                'variables': f'{{"videoId":{video_id}}}',
+                'query': self._VIDEO_VIEW,
+            }, headers={
+                'content-type': 'application/json',
+                'x-platform-id': 'web',
+                'x-device-id': 'desktop',
+                'x-client-version': '2024.12-5',
+            })['data']['video']
 
         formats = []
-        security = self._download_json(
-            'https://playback.video.globo.com/v2/video-session', video_id, f'Downloading security hash for {video_id}',
-            headers={'content-type': 'application/json'}, data=json.dumps({
-                'player_type': 'desktop',
+        video = self._download_json(
+            'https://playback.video.globo.com/v4/video-session', video_id,
+            f'Downloading resource info for {video_id}',
+            headers={'Content-Type': 'application/json'},
+            data=json.dumps({
+                'player_type': 'mirakulo_8k_hdr',
                 'video_id': video_id,
                 'quality': 'max',
                 'content_protection': 'widevine',
-                'vsid': '581b986b-4c40-71f0-5a58-803e579d5fa2',
-                'tz': '-3.0:00',
+                'vsid': f'{uuid.uuid4()}',
+                'consumption': 'streaming',
+                'capabilities': {'low_latency': True},
+                'tz': '-03:00',
+                'Authorization': try_get(self._get_cookies('https://globo.com'),
+                                         lambda x: f'Bearer {x["GLBID"].value}') or '',
```

Comment:
weird that this is in the payload instead of the headers?

but anyways, maybe call `filter_dict` on the whole payload dict to just remove `Authorization` if the value is `None`?